src/components/Button/Button.module.css

```diff
@@ -10,7 +10,6 @@
  * 3) Margin 0 added becuase Safari has margin on the button
  */
 .button {
-  @mixin eds-theme-typography-button;
```

Comment:
Removed this line because there is now a default size and the sizes all specify the typography styles.